ompi/mca/osc/monitoring/Makefile.am

```diff
@@ -17,7 +17,8 @@ monitoring_sources  = \
 	osc_monitoring_active_target.h \
 	osc_monitoring_dynamic.h \
 	osc_monitoring_module.h \
-	osc_monitoring_template.h
+	osc_monitoring_template.h \
+	osc_monitoring_template_gen.h
```

Comment:
template_gen.h shouldn't be in monitoring_sources.  Since it's generated at every run of configure, it shouldn't be distributed in the tarball (which monitoring_sources will be).  You probably need to add a nodist_mca_osc_monitoring_la_SOURCES line (I think that's right, my Automake foo is getting rusty).

Reply:
Done in commit 2011660bea186e303a9f46f5591d19838343abc2 .
I'm not a automake guru, but it seems like adding osc_monitoring_template_gen.h to the nodist_mcacomponent_HEADERS target worked as intended.

Reply:
yeah, I saw that in the next commit (I review commits independently, because changes should be bisectable).  You need to merge those two commits into one before merging this PR to master.